client/webpack/extensionConfig.js

```diff
@@ -62,15 +62,15 @@ module.exports = (dir) =>
 				],
 			}),
 		],
-		optimization: {
-			splitChunks: {
-				cacheGroups: {
-					vendors: {
-						test: /[\\/]node_modules[\\/]/,
-						name: 'vendors',
-						chunks: 'all',
-					},
-				},
-			},
-		},
+		// optimization: {
+		// 	splitChunks: {
+		// 		cacheGroups: {
+		// 			vendors: {
+		// 				test: /[\\/]node_modules[\\/]/,
+		// 				name: 'vendors',
+		// 				chunks: 'all',
+		// 			},
+		// 		},
+		// 	},
+		// },
```

Comment:
Somehow this option prevents connection from the extension to service worker so we had to comment it out

Reply:
I'll check it and update later